code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -212,6 +212,9 @@
 	. = ..()
 	if(methods & TOUCH)
 		exposed_mob.extinguish_mob() // extinguish removes all fire stacks
+	if((methods & VAPOR) && isfelinid(exposed_mob))
```

Comment:
I'm not very well versed in /tg/ code, but from what i've heard from a friend "VAPOR" is also used for smoke machines and possibly water vapor, which could result in a stunlock

Reply:
actually its foam
which is even stronger for things like grenades but cant get smoke machined as easily
so less grief, more powergame